sources/styles.css

```diff
@@ -484,3 +484,104 @@ section.about-container h4 {
   color: #344563;
   margin-bottom: 80px;
 }
+
+/* contact start */
+footer.contact-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 60px;
+  margin-left: 24px;
+  margin-bottom: 60px;
+  background-image: url("../images/illustration-contact-form.svg");
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position-x: 100%;
+  background-position-y: 77%;
+}
+
+footer.contact-container h3 {
+  text-align: center;
+  width: 327px;
+  height: 363px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+ul.form-input {
+  list-style-type: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+ul.form-input .input {
+  width: 327px;
+  height: 48px;
+  border-radius: 4px;
+  border: 1px solid #d0d9d4;
+  margin: 24px;
+  padding: 13px 15px;
+}
+
+.text-area {
+  width: 327px;
+  height: 114px;
+  border-radius: 4px;
+  margin: 24px;
+  padding: 12px;
+}
```

Comment:
- [ ] Don`t you think, that if you will decrease the value of margin of each input in the form, your contact form will be closer to the Figma template?

Reply:
![Capture](https://user-images.githubusercontent.com/90656697/149029030-524f811b-b2dd-4b99-83d3-b216c76d6ade.JPG)

hey @elmar8287 , thank you for your review. I went ahead and fixed the margins and centered my page but for this comment where you told me to decrease margins, im actually using 24px as is stated in the figma design file as you can notice in the screenshot above. Do you still want me to change it?